src/lib/handle_ping_start.go

```diff
@@ -0,0 +1,86 @@
+package lib
+
+import (
+	"log"
+	"sync"
+	"time"
+
+	"github.com/zairza-cetb/bench-routes/src/lib/filters"
+	"github.com/zairza-cetb/bench-routes/src/lib/modules/ping"
+	"github.com/zairza-cetb/bench-routes/src/lib/utils"
+	"github.com/zairza-cetb/bench-routes/tsdb"
+)
+
+func handlePingStart(config utils.YAMLBenchRoutesType, pingServiceState string) bool {
+	if pingServiceState == "passive" {
+
+		Configuration.Config.UtilsConf.ServicesSignal.Ping = "active"
+		_, e := Configuration.Write()
+		if e != nil {
+			panic(e)
+		}
+		go func() {
+			pingConfig := Configuration.Config.Routes
+			pingInterval := getInterval(Configuration.Config.Interval, "ping")
+			if pingInterval == (testInterval{}) {
+				log.Fatalf("interval not found in configuration file for ping")
```

Comment:
It would be really good if the function which runs as a different go-routine _(the one selected)_ is kept in this file along with `doPing()` since the newer go-routine is that part in `handlers.go` which increases the complexity.

I would suggest making a newer directory named `handlers`, and keep this file in it. which would contain the code belonging to the function running as an independent go-routine.